app/assets/javascripts/initializers/initializePodcastPlayback.js

```diff
@@ -183,17 +183,30 @@ function initializePodcastPlayback() {
     Array.prototype.forEach.call(records, function (record) {
       var episodeSlug = record.getAttribute('data-episode');
       var podcastSlug = record.getAttribute('data-podcast');
-      record.onclick = function () {
-        if (podcastBarAlreadyExistAndPlayingTargetEpisode(episodeSlug)) {
+      let isPressed = record.getAttribute('aria-pressed');
+
+      var togglePodcastState = function (e) {
+        e.preventDefault();
+        var isValidKey = e.type === "keydown" && ["32", "13"].includes(e.keyCode);
+        
+        if (
+            (isValidKey || e.type === "onclick") 
+            && podcastBarAlreadyExistAndPlayingTargetEpisode(episodeSlug)
+          ) {
+          
           var audio = getById('audio');
+          record.setAttribute('aria-pressed', String(!isPressed));
+
           if (audio) {
             playPause(audio);
           }
         } else {
           stopRotatingActivePodcastIfExist();
           loadAndPlayNewPodcast(episodeSlug);
         }
-      };
+      }
+      record.onclick = togglePodcastState;
```

Comment:
Refactor (blocking): Attach the event listeners via [addEventListener](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener).